examples/queue-manager-demo/package.json

```diff
@@ -12,7 +12,7 @@
   },
```

Comment:
You can use `PendingSwapNetworkStatus` from `rango-types` in `examples/queue-manager-demo/src/flows/rango/types.ts`